crates/bevy_ui/src/node_bundles.rs

```diff
@@ -50,6 +50,11 @@ pub struct NodeBundle {
     pub computed_visibility: ComputedVisibility,
     /// Indicates the depth at which the node should appear in the UI
     pub z_index: ZIndex,
+    /// The node's position in the UiStack. Nodes with lower stack indices are drawn first.
+    /// Nodes with a higher stack index are drawn over nodes with a lower stack index.
+    ///
+    /// This component is automatically managed by `ui_stack_system`.
```

Comment:
Link would be nice here.

Reply:
Ditto replicated doc comments before.

Reply:
Ah, can't link to `ui_stack_system` because it's private, that's why the CI was failing.